nautobot/extras/api/nested_serializers.py

```diff
@@ -93,6 +93,23 @@ class Meta:
         fields = ["id", "url", "name", "image"]
 
 
+class NestedJobLogEntrySerializer(serializers.ModelSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name="extras-api:joblogentry-detail")
+
+    class Meta:
+        model = models.JobLogEntry
+        fields = [
+            "id",
+            "url",
+            "absolute_url",
```

Comment:
Is `absolute_url` the URL of the `log_object`? I see that it's `null` in all of your example output in this PR, so I wasn't sure.

Reply:
These are used for list view of log entries in the JobResult detail view, if and only if `log_object` is also populated.